src/Workspaces/Core/Portable/Workspace/Host/Mef/MefHostServices.cs

```diff
@@ -126,7 +124,15 @@ private static ImmutableArray<Assembly> LoadDefaultAssemblies()
                 "Microsoft.CodeAnalysis.VisualBasic.Features"
             };
 
-            return LoadNearbyAssemblies(assemblyNames);
+        internal static bool IsDefaultAssembly(Assembly assembly)
+        {
+            var name = assembly.GetName().Name;
```

Comment:
>GetName() [](start = 32, length = 9)

I believe GetName allocates a new name every time it's called. It'd be probably better to do `assembly.FullName.StartsWith(...)`.

Reply:
This should be called very rarely - only once per every new language that we want to compute the serializable options. We can tune this if turns out to be a performance problem.